lib/ReactViews/Analytics/PolygonParameterEditor.jsx

```diff
@@ -38,7 +38,7 @@ const PolygonParameterEditor = React.createClass({
     },
 
     getValue() {
-        const pointsLongLats = this.props.parameterValues[this.props.parameter.id];
+        const pointsLongLats = this.props.parameterValues[this.props.parameter.id][0];
```

Comment:
Another piece of this puzzle is that the defaultValue for a parameter is not actually getting set, ever, in the code you're looking at. I've fixed this in https://github.com/TerriaJS/terriajs/pull/1910.